src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Figure {
+    abstract void getInfoAboutFigure();
+}
```

Comment:
I think Figure sounds like an abstract class, doesn't it?
```suggestion
public interface Drawable {
    void draw();
}
```

Reply:
If i make figure class abstarct than tests are not passing:
Results :
Failed tests:   programStructureTest(core.basesyntax.StructureTest): You should have only one common superclass for your figure classes
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0